pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -147,67 +147,47 @@ func watchChaosResources(clientSet client.Client, c controller.Controller) error
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	reqLogger := chaosTypes.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.Info("Reconciling ChaosEngine")
+	reqLogger := startReqLogger(request)
 	err := r.getChaosEngineInstance(request)
 	if err != nil {
 		if k8serrors.IsNotFound(err) {
 			// Request object not found, could have been deleted after reconcile request.
 			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
 			// Return and don't requeue
-			r.recorder.Eventf(engine.Instance, corev1.EventTypeWarning, "ChaosResourcesOperationFailed", "Unable to find chaosengine")
+			//r.recorder.Eventf(engine.Instance, corev1.EventTypeWarning, "ChaosResourcesOperationFailed", "Unable to find chaosengine")
```

Comment:
Why did you comment this?

Reply:
Its not required @ispeakc0de .
If engine is unavailable in the cluster, then this operator will go in CrashBackLoopOff`.
Please add `comment`, rather than `changes requested` for the issues you are not confirmed on.